packages/legend-query-builder/style/shared/_query-loader.scss

```diff
@@ -0,0 +1,293 @@
+    height: 4.4rem;
+    position: relative;
+
+    &__input {
+      font-size: 1.3rem;
+      height: 2.8rem;
+      padding-right: 2.8rem;
+      padding-left: 1rem;
+      width: 100%;
+      background: var(--color-dark-grey-85);
+      border-radius: 0.2rem;
+      border: 0.1rem solid var(--color-dark-grey-200);
+
+      &__container {
+        @include flexCenter;
+
+        position: relative;
+        height: 2.8rem;
+        width: 100%;
+      }
+
+      &--searching {
+        padding-right: 5.6rem;
+      }
+
+      &__search__icon {
+        @include flexCenter;
+
+        position: absolute;
+        right: 0.2rem;
+        height: 2.8rem;
+        width: 2.8rem;
+
+        svg {
+          color: var(--color-light-shade-100);
+        }
+      }
+
+      &__clear-btn {
+        @include flexCenter;
+
+        position: absolute;
+        right: 0.2rem;
+        height: 2.8rem;
+        width: 2.8rem;
+
+        svg {
+          color: var(--color-dark-grey-400);
+        }
+
+        &:hover svg {
+          color: var(--color-dark-grey-500);
+        }
+      }
+    }
+  }
+
+  &__body {
+    padding: 0;
+    margin-top: 2rem;
+    margin-bottom: 4rem;
+    height: 40rem;
+    overflow-y: overlay;
+  }
+
+  &__action-btn {
+    height: 1.7rem;
+    font-size: 1.2rem;
+    display: flex;
+    padding: 0.2rem;
+    margin-right: 0.2rem;
+  }
+
+  &__result__section {
+    width: 57rem;
+    max-height: 5.5rem;
+    border: 0.1rem solid var(--color-dark-grey-200);
+    background: var(--color-dark-grey-100);
+    display: flex;
+    margin-bottom: 1rem;
+    cursor: pointer;
+  }
+
+  &__result__section:hover {
+    background: var(--color-dark-grey-200);
+  }
+
+  &__result__content__section {
+    text-align: start;
+  }
+
+  &__result__title {
+    padding: 1rem 1rem 0;
+    color: var(--color-light-grey-300);
+    font-weight: 500;
+  }
+
+  &__result__title__editor {
+    display: flex;
+
+    &__input {
+      width: 100%;
+      min-width: 35rem;
+      margin-left: 1rem;
+      margin-top: 0.5rem;
+      padding: 0.5rem;
+      border-radius: 0.1rem;
+      background: var(--color-dark-grey-50);
+      color: var(--color-light-grey-200);
+      height: 2.5rem;
+      overflow-x: overlay;
+      border: 0.1rem solid var(--color-dark-grey-300);
+
+      &:focus {
+        border: 0.1rem solid var(--color-blue-200);
+        outline: none;
+      }
+    }
+
+    &__actions {
+      display: flex;
+    }
+
+    &__save-btn,
+    &__cancel-btn {
+      font-size: 1.2rem;
+      height: 2.2rem;
+      padding: 1rem;
+      margin-top: 0.8rem;
+      margin-left: 0.55rem;
+
+      &[disabled] {
+        cursor: not-allowed;
+      }
+    }
+
+    &__cancel-btn {
+      background: var(--color-dark-grey-100);
+      border: 0.1rem solid var(--color-dark-grey-300);
+
+      &:hover {
+        background: var(--color-dark-grey-250);
+      }
+    }
+  }
+
+  &__result__description {
+    padding: 1rem;
```

Comment:
I think perhaps `@include flexCenter;`might be good here for aligning icons and text in the description labels